### PR TITLE
refactor(Tactics): Start of PEP8 compliance for Tactics

### PR DESCRIPTION
**Pull request by david2194**

### CODE REVIEW

File: ai/STA/Strategy/DefenseWallNoKick.py [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
-from RULEngine.Game.OurPlayer import OurPlayer
-from ai.Algorithm.evaluation_module import closest_players_to_point, Pose, Position
-from ai.STA.Tactic.AlignToDefenseWall import AlignToDefenseWall
-from ai.STA.Tactic.GoalKeeper import GoalKeeper
-from ai.STA.Tactic.go_kick import GoKick
-from ai.STA.Tactic.position_for_pass import PositionForPass
-from ai.STA.Tactic.tactic_constants import Flags
+from ai.Algorithm.evaluation_module import Pose, Position
+from ai.STA.Tactic.align_to_defense_wall import AlignToDefenseWall
+from ..Tactic.GoalKeeper import GoalKeeper
```
Comment (PhiBabin) [inline ai/STA/Strategy/DefenseWallNoKick.py]: Relative import

File: ai/STA/Tactic/go_to_position_no_pathfinder.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from RULEngine.Util.Pose import Pose
 from ai.states.game_state import GameState
 from .Tactic import Tactic
-from . tactic_constants import Flags
+from .tactic_constants import Flags
```
Comment (PhiBabin) [inline ai/STA/Tactic/go_to_position_no_pathfinder.py]: relative import

File: ai/STA/Tactic/rotate_around_ball.py [hunk 3]
```diff
@@ -20,8 +20,5 @@ def __init__(self, game_state: GameState, player: OurPlayer, target: Pose, args:
     def exec(self):
```
Comment (PhiBabin) [inline ai/STA/Tactic/rotate_around_ball.py]: Il faudrait spliter les tactics stateless et les statics avec un state dans d'un catégorie. Parce que le fait qu'ils n'ont pas la même behavior est un peu mélangeant. Une tactic stateless va avoir un overload de exec et donc devra appelé .exec(). Une tactic avec un state doit renvoyer l'action sans appelé .exec(). On peut mettre cette critique dans le TODO de STA 2.0 #207 .

**Comment by david2194** [comment]
Do not merge now. This PR might break the project for windows users since it renames a couple of files to remove random capitalization, which is not handled properly in windows. In addition, we will need to do the same stuff for Strategy and Actions. We will merge once all three modules are done

**Comment by david2194** [comment]
Is ready and all tests are passing. However, will need to test real-life and in simulation because my Linux dual-boot is not up and running yet (and @PhiBabin gave up on the new grSim for windows)

**Comment by sim0504** [comment]
If you wanted to be PEP8, do it all the way :)

> Python coders from non-English speaking countries: please write your comments in English, unless you are 120% sure that the code will never be read by people who don't speak your language.
[https://www.python.org/dev/peps/pep-0008/#comments](url)

**Pull request merged**